### ISSUE TITLE
My Web Archive pages don't always show up

### ISSUE THREAD

**Issue by cgorringe** [internetarchive/wayback-machine-webextension#876]
**Describe the bug**

Report:
"I manually saved a page with the My Web Archive button ticked and I can see it there, but the pages saved via the extension haven’t showed up there yet, despite the fact it shows it has detected an account and the option is enabled."

**Additional context**

Likely a bug on the website / backend and not in the extension. Needs further testing.

**Comment by KGeetings** [comment]
This appears to still be an issue. I was wondering if you had done any further testing to figure out where the error is occurring. While I'm not the original reporter, I would gladly provide any information necessary to help debug this. 

Currently running the latest version of the extension (3.2), on the latest Firefox (114.0.1), with the latest version of Windows 11 (22H2/22631.1835). This appears to be happening on all of my computers, so it's not an isolated incident. I have signed out and back in several times to see if that would solve the issue, as well as made sure the "save to my web archive" is always selected.

**Comment by cgorringe** [comment]
Thanks for reporting this @KGeetings - it does appear to be an issue still.

**Comment by anishsarangi** [comment]
I was testing this issue but it is working for me. Not sure if it is specific to any website.

**Comment by KGeetings** [comment]
Hello @anishsarangi at least for me, this appears to be an issue across all websites.

**Comment by jody-frankowski** [comment]
I have the same problem, with both the webui and the extension. With the webui I can see that I get a `429 Too Many Requests` for the POST on `__wb/web-archive/`.
I would guess that it's an IP reputation issue, but I have tried with and without my VPN and I have the same problem in both cases, so I'm not sure.

**Comment by VishnuRaghavan** [comment]
Could anybody figure out this is a backend or front-end issue yet ?

**Comment by jody-frankowski** [comment]
@VishnuRaghavan Isn't this a backend issue, based on the HTTP error?
Moreover, the front-end (web and extension), should warn the user that there was an issue with the submission.
An additional improvement would be to have an automatic retry in case there is a temporary backend issue.

**Comment by VishnuRaghavan** [comment]
> @VishnuRaghavan Isn't this a backend issue, based on the HTTP error? Moreover, the front-end (web and extension), should warn the user that there was an issue with the submission. An additional improvement would be to have an automatic retry in case there is a temporary backend issue.

Yes i have noticed that its a backend issue, where the job takes more time to save the captures. Thus it takes time to reflect on extension as well as the web version. We need to either reduce the load some how or expose api's to give job status / progress in real time (or polling). Or We could simply warn the user like 'WM take couple of minutes to save captures' - something like that.

**Comment by jody-frankowski** [comment]
> Yes i have noticed that its a backend issue, where the job takes more time to save the captures. Thus it takes time to reflect on extension as well as the web version.

Do you mean saving the capture in "My Archive"? Because as of today, none of my saves (with the extension or the webui) have showed up in "My Archive".

**Comment by VishnuRaghavan** [comment]
Nothing shows up in Web Archives section of web version for me too. I meant that the webpages are saved to the backend after a delay, ie; you could see it on the calendar section.
